front-end/src/app/shared/services/validate.service.spec.ts

```diff
@@ -60,14 +60,19 @@ describe('ValidateService', () => {
     let result: ValidationErrors | null = validator(service.formValidatorForm.get('change_of_address') as FormControl);
     expect(result).not.toBe(null);
     if (result) {
-      expect(result['pattern'].requiredPattern).toBe('must be boolean,null');
+      expect(result['pattern'].requiredPattern).toBe('Allowed values: X, ');
```

Comment:
So even thought we merged the updates into the validate repo, we want to point to the old specs?

Reply:
if we point to a later commit in fec-validate's history (HEAD of develop for example) this will break

Reply:
Yes, good catch. My local validate repo got behind. I updated both the fecfile-web-app and fecfile-web-api repos to point back to the develop branch.